cdap-distributions/src/etc/cdap/conf.dist/cdap-env.sh

```diff
@@ -38,6 +38,8 @@ LOCAL_DIR="/var/tmp/cdap"
 # temporary storage for Apache Twill
 TEMP_DIR="/tmp"
 
+HEAPDUMP_ON_OOM=true
```

Comment:
@dereklwood @mattwuenschel I'm not sure why some variables here have `export` and others don't... Any suggestion?